tests/foreman/ui/test_contentview.py

```diff
@@ -1559,13 +1680,37 @@ def test_positive_remove_published_custom_ostree_version(self):
         @Assert: Published version with OStree contents should be removed
         successfully.
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        org = entities.Organization().create()
+        prod = entities.Product(organization=org).create()
+        # Creates new ostree repository using api
+        ostree_repo = entities.Repository(
+            content_type='ostree',
+            url=FEDORA23_OSTREE_REPO,
+            product=prod,
+            unprotected=False,
+        ).create()
+        ostree_repo.sync()
+        cv = entities.ContentView(
+            name=gen_string('alpha'),
+            organization=org,
+        ).create()
+        cv.repository = [ostree_repo]
+        cv.update(['repository'])
+        cv.publish()
+
+        cv_info = cv.read_json()['versions'][0]
```

Comment:
it is old format, please use read() instead (e.g. 'cvv = content_view.version[0].read()')